src/components/Search/CreateIncident.js

```diff
@@ -3,23 +3,31 @@ import { connect } from 'react-redux'
 import { TextField } from 'material-ui'
 import FlatButtonStyled from '../elements/FlatButtonStyled'
 import { updateIncidentCreationInput } from '../../actions/incidentActions'
-const onSubmit = (input, ticketSystem, history) => () => {
 
-    history.push(/tickets/ + input)
+const onSubmit = (input, history) => () => {
+    if (input)
+    {history.push(/tickets/ + input)}
```

Comment:
Please align the opening curly brace with the of statement on the previous line and indent this statement.

Reply:
@strand Fixed!